### PR TITLE
Update

### PR DESCRIPTION
**Pull request by eneufeld**

### CODE REVIEW

File: backend/plugins/org.eclipse.emfcloud.coffee.modelserver.feature/feature.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      label="Coffee Modelserver Feature"
+      version="0.1.0.qualifier"
+      provider-name="EclipseSource">
+
+   <description url="https://github.com/eclipse-emfcloud/coffee-editor">
+      Coffee Modelserver Application, for creating and editing workflow
+diagrams.
+   </description>
+
+   <copyright url="https://github.com/eclipsesource/volkswagen">
```
Comment (sgraband) [inline backend/plugins/org.eclipse.emfcloud.coffee.modelserver.feature/feature.xml:13]: I would guess that this is a wrong URL, right?

File: backend/plugins/org.eclipse.emfcloud.coffee.workflow.glsp.server.feature/feature.xml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      label="Workflow GLSP Server Feature"
+      version="0.1.0.qualifier"
+      provider-name="EclipseSource">
+
+   <description url="https://github.com/eclipse-emfcloud/coffee-editor">
+      Workflow GLSP Server Application, for creating and editing workflow
+diagrams.
+   </description>
+
+   <copyright url="https://github.com/eclipsesource/volkswagen">
```
Comment (sgraband) [inline backend/plugins/org.eclipse.emfcloud.coffee.workflow.glsp.server.feature/feature.xml:13]: Same here

**Review by sgraband — COMMENTED** [review]
Thanks for the changes @eneufeld!

One general remark that i have is that we probably should update the copyright headers to contain the year 2022, right? Or is 2021 fine, since the changes were commited to branches last year?

Additionally i found some (smaller) features that are currently not working. (I am fine with not fixing them now, but then we should probably create tickets for them.)
- Tree View:
    - Deletion of nodes/flows does not work and will "break" the modelserver.
    - RAM clock speed and size are not displayed anymore if you reopen the tree view.
- The marquee tool in the tool palette is not working. We should either remove it from the palette or fix its behavior.
- The assertions in the textual dsl are not validated at any point i believe.
- The link to the diagram editor on the getting started page is not working.

Other than that the changes look good to me and the editor still works as expected.

**Comment by eneufeld** [comment]
I take a look to fix some low hanging fruits. For everything else I will create a follow up. I will fix the headers asap.

**Review by sgraband — APPROVED** [review]
LGTM :+1:

**Pull request merged**